.devops/nix/package.nix

```diff
@@ -222,8 +225,8 @@ effectiveStdenv.mkDerivation (
     # TODO(SomeoneSerge): It's better to add proper install targets at the CMake level,
     # if they haven't been added yet.
     postInstall = ''
-      mv $out/bin/main $out/bin/llama
-      mv $out/bin/server $out/bin/llama-server
+      mv $out/bin/main${executableSuffix} $out/bin/llama
```

Comment:
Do we really want to remove the `.exe` when targeting windows?

Reply:
Valid point - we would probably like to have the `executableSuffix` on the right side of the copy too.